### DISCUSSION TITLE
Upvote BullMQ for priority on Cloudflare workers!

### DISCUSSION THREAD

**Discussion by prescience-data** [taskforcesh/bullmq#478] [Ideas]
Cloudflare Workers have just added support for Node and are taking votes on which packages to prioritize.

https://workers.cloudflare.com/node
https://blog.cloudflare.com/node-js-support-cloudflare-workers/

Let's get BullMQ trending! 🔥📈

**Comment by manast** [comment]
Great!. Since Bull/BullMQ is still not in the list we need to add the URL to the package manually here:
https://airtable.com/shrM9MdUR8ojbUdSZ

and paste https://www.npmjs.com/package/bull 
and https://www.npmjs.com/package/bullmq

**Comment by jpmelnik** [comment]
Can I use BullMQ with Cloudflare Workers?

**Reply by manast** [reply]
In theory when Upstash fully supports BullMQ it would work: https://developers.cloudflare.com/workers/databases/native-integrations/upstash/
